core/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java

```diff
@@ -155,8 +157,10 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.startObject();
             if (item.isFailure()) {
                 ElasticsearchException.renderThrowable(builder, params, item.getFailure());
+                builder.field("status", ExceptionsHelper.status(item.getFailure()).getStatus());
             } else {
                 item.getResponse().toXContent(builder, params);
+                builder.field("status", RestStatus.OK.getStatus());
```

Comment:
`SearchResponse` implements `StatusToXContent`so you can use its `status()` method instead of forcing `OK`.

Something like: 
```java
SearchResponse response = item.getResponse();
builder.field(Fields.STATUS, response.status());
response.toXContent(builder, params);
```